packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -114,10 +122,17 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
   }
 
   async syncGateways() {
-    const gws = await this.client.listGateways();
-    this.setState(draft => {
-      draft.gateways = new Map(gws.map(g => [g.uri, g]));
-    });
+    try {
+      const gws = await this.client.listGateways();
+      this.setState(draft => {
+        draft.gateways = new Map(gws.map(g => [g.uri, g]));
+      });
+    } catch (error) {
+      this.notificationsService.notifyError({
+        title: 'Could not fetch databases',
```

Comment:
This name is okay for now but we should change it once we start using gateways for something else than just databases.

Though, with the recent changes to `tsh proxy db`, Zac mentioned how we can consider to stop using "gateway" as a concept and instead just execute `tsh proxy db` underneath. The implementation of it is still unclear to me, but I should know more after I work on the changes to proxies that Roman introduced.

Reply:
I understand, I added it here because this method is used in two places, so I wanted to avoid duplication of error messages.